### ISSUE TITLE
Occurence support in objects doesn't support max values

### ISSUE THREAD

**Issue by itamarst** [anweiss/cddl#128]
It is my understanding (and I may be misinterpreting the spec) that I can say "this object can have up to 3 string keys" with the following syntax variations:

```
limited = { *3 tstr => tstr }
```

or

```
limited = { *3 my_key => tstr }
my_key = tstr
```

If I try to validate the following:

```json
{"A": "B"}
```

the CLI (0.9.0-beta1) fails with `[ERROR] Validation of "onekey.json" failed: error validating at the root of the JSON document: expecting object value of type tstr, got object`.

However, if I change the schemas to have unlimited length, e.g.

```
limited = { * tstr => tstr }
```

then it validates just fine.

I have not tested min values (e.g. `1*3` or `1*`) but I imagine it might have similar issues.

**Comment by anweiss** [comment]
Thanks for reporting @itamarst. This is being fixed in #138. This was never actually implemented for whatever reason.

**Comment by anweiss** [comment]
Implemented in #138. Closing
